### ISSUE TITLE
Color calibration module gives hint "double CAT applied" but there is only one module instance

### ISSUE THREAD

**Issue by cytrinox** [darktable-org/darktable#7286]
**Describe the bug**
Migrated today to latest darktable master 3.3.0~git2079.19b17c9cc. When I open any of my already imported CR2 raws from Darktable 3.2, I get the "double CAT applied" hint.

**To Reproduce**
1. Change chromatic adaption defaults to **modern**
2. Restart darktable
3. Choose a unprocessed CR2 file in lighttable and open it with double click in darkroom.
4. Color calibration shows the "double CAT applied" hint, even if no other instance is active
5. Switch back to lighttable and reopen image again
6. The warning disappears.

You could instead after step 4 create a duplicate or original duplicate and switch between the two, the warning also disappears.

**Expected behavior**
No warning at all after first open.

**Screenshots**
![Screenshot from 2020-12-12 16-17-06](https://user-images.githubusercontent.com/509535/101988479-21861e80-3c9a-11eb-82a0-9e75b72f88ae.png)

Platform (please complete the following information):

    Darktable Version: 3.3.0~git2079.19b17c9cc.
    OS: Debian Bullseye
    OpenCL: yes
    PNY Nvidia Quadro 450.80.02-1

Binary package was obtained from https://software.opensuse.org/download.html?project=graphics:darktable:master&package=darktable

**Comment by cytrinox** [comment]
@aurelienpierre here is the CC bug report. It even happens on a fresh file import.

**Comment by johnny-bit** [comment]
Looks like duplicate of #7271
